pkg/clients/grpc/connect.go

```diff
@@ -142,7 +142,7 @@ func connect(cfg Config) (*grpc.ClientConn, security, error) {
 	tc := insecure.NewCredentials()
 
 	if cfg.AttestedTLS {
-		err := ReadManifest(cfg.Manifest, &attestationConfiguration)
+		err := ReadManifest(cfg.BackendInfo, &attestationConfiguration)
```

Comment:
I do not get this... You use function `ReadManifest()` to read `BackendInfo`? It is confusing.